docs/develop/dotnet/failure-detection.mdx

```diff
@@ -125,6 +125,21 @@ return await Workflow.ExecuteActivityAsync(
     });
 ```
 
+### Overriding the retry interval with Next Retry Delay {#next-retry-delay}
+
+You may throw an [Application Failure](/references/failures#application-failure) with the NextRetryDelay field set. This value will replace and override whatever the retry interval would be on the retry policy.
+
+For example, if in an activity, you want to base the interval on the number of attempts, you might do:
```

Comment:
```suggestion
For example, you might scale the next Retry delay interval based on the current number of attempts.
Here's how you'd do that in an Activity:
```

- Activity is a reserved term, so capitalized

Reply:
This is the form already present/approved at `docs/develop/java/failure-detection.mdx` and may be a rule worth applying consistently there too instead of just here (but feel free to apply here first)